### PR TITLE
Clean up "The List"

### PR DESCRIPTION
**Pull request by dragonmaus**
Cleaned things up, did a little reorganising, removed some blatantly nonviable rules, and made things overall much more consistent.

Also made it a markdown document, to allow some minimal formatting.

### CODE REVIEW

**Comment by a-random-lemurian** [comment]
Please maintain the normal spelling used by the original Discord message. If the message used American English and it became a rule, the rule should remain in American English spelling.

File: First Last Rules.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+1. First Last is not allowed to own a Marauder Bactrian.
+1. First Last is not allowed to recreate the United States of America within human space.
+1. First Last is not allowed to place any system within jump drive range of the Deep.
+1. First Last is not allowed to seduce any non-human being.
+1. First Last is not allowed to play Fleetwood Mac's "The Chain" at full volume on the bridge of his flagship.
+1. First Last is not allowed to purchase "18 tonnes of cargo, 12 tonnes of outfits, and seven navy cruisers".
+1. First Last is not allowed to say "get that sussy baka!" while referring to anyone on board one of his ships.
+1. First Last is not allowed to name any planet after any female politician born between the years 1900 and 2100.
+1. First Last is not allowed to create a JavaScript framework.
+1. First Last is not allowed to "weaponise the backrooms".
```
Comment (a-random-lemurian) [inline First Last Rules.md:180]: Original message used American English spelling (weaponize).

Comment (dragonmaus) [reply First Last Rules.md:180]: Sorry, that one slipped by. I did try to normali**z**e on American spellings. 😬

File: First Last Rules.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+1. First Last is not allowed to express his political beliefs.
+1. First Last is not allowed to express his religious beliefs.
+1. First Last is not allowed to express anything he believes in in any capacity.
+1. First Last is not allowed to refer to Katya as "Hurricane Katyna".
+1. First Last is not allowed to tribute non-human worlds.
+1. First Last is not allowed to tribute monks.
+1. First Last is not allowed to refer to any Southern ship designers using any name involving a gaseous chemical and the word "pocket".
+1. First Last is not allowed to use computers, ships, or any motorized vehicle on Harmony, Girtab.
+1. First Last is not allowed to own a Marauder Fury.
+1. First Last is not allowed to seduce an AI to do his bidding.
```
Comment (a-random-lemurian) [inline First Last Rules.md:170]: Original rule used gender-neutral pronouns.

Comment (dragonmaus) [reply First Last Rules.md:170]: The majority of the rules (as well as the general vernacular within the community) used masculine pronouns for First Last, so I normalized on that.

Comment (a-random-lemurian) [reply First Last Rules.md:170]: Alright, but all of my rules used gender-neutral pronouns.

I'll let it slide this time, but please next time keep rules as they are, preferably change the capitalization of some letters, and only reword if necessary.

Additionally, maintain the English variant of the submitted rule, and if it uses gender neutral pronouns for First Last, keep it that way.

**Comment by a-random-lemurian** [comment]
Anyways, let's merge this PR now.

**Pull request merged**